e2e/keywords/volume.resource

```diff
@@ -10,9 +11,19 @@ Create a volume ${size} GB with ${replica_count} replicas
     Set Test Variable    ${volume_name}
 
 Write data to the volume
-    ${volume_data_checksum} =     write_volume_random_data   ${volume_name}    1024
+    ${volume_data_checksum} =     write_volume_random_data   ${volume_name}    2048
     Set Test Variable    ${volume_data_checksum}
 
+Delete replica to trigger replica rebuilding
```

Comment:
Rename to `Delete a replica not on the volume node to trigger replica redbuild`

Reply:
After modification the test steps would become:

* Delete a replica **not on the volume node** to trigger replica rebuilding
* During replica rebuilding, reboot **replica node**

which is inconsistent.

I'll make it:
* Delete a replica **on replica node** to trigger replica rebuilding
* During replica rebuilding, reboot **replica node**